tasmota/i18n.h

```diff
@@ -70,7 +70,7 @@
 #define D_JSON_ERROR "Error"
 #define D_JSON_EVENT "Event"
 #define D_JSON_EVERY "Every"
-#define D_JSON_EXPORT "Export"
```

Comment:
This Shouldn't be deleted.

Reply:
Entry is double in List

look at Line -214 #define D_JSON_EXPORT "Export"